.pre-commit-config.yaml

```diff
@@ -8,4 +8,4 @@ repos:
     hooks:
     -   id: mypy
         args: [--strict, --ignore-missing-imports]
-        additional_dependencies: [types-requests==2.31.0]
\ No newline at end of file
+        additional_dependencies: [types-requests==2.31.0, pydantic==2.5.3]
```

Comment:
no worries about not needing a caret here -- thanks for including this!

Reply:
no worries 👍